web/src/message_view.ts

```diff
@@ -303,7 +334,18 @@ export function try_rendering_locally_for_same_narrow(filter, opts) {
     return true;
 }
 
-export function show(raw_terms, show_opts) {
+type ShowOpts = {
```

Comment:
ShowMessageViewOpts or similar is a better name here.

Reply:
👍 went with `ShowMessageViewOpts`